gobblin-core/src/main/java/org/apache/gobblin/publisher/BaseDataPublisher.java

```diff
@@ -122,14 +125,14 @@
   static final String PUBLISH_RETRY_ENABLED = DATA_PUBLISHER_RETRY_PREFIX + "enabled";
 
   static final Config PUBLISH_RETRY_DEFAULTS;
-  protected final Config retrierConfig;
+  protected final Config retryerConfig;
```

Comment:
wow, really hard to know how to spell this!  I might prefer slightly the form you just chose, although I do find this in the AWS SDK - https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/stepfunctions/builder/states/Retrier.html

I guess we just want it consistent in the codebase

Note though that the guava lib has a `RetryerBuilder` and the file at the end of this PR is `RetryerFactory`, so seemingly where the momentum's going

Reply:
haha. I think we need to follow https://github.com/rholder/guava-retrying/blob/master/src/main/java/com/github/rholder/retry/Retryer.java name

Reply:
fo' sho'!